<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>BUAABrowser</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logos.qrc">
    <normaloff>:/BrowserLogo.ico</normaloff>:/BrowserLogo.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(102,157,190);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="tabsClosable">
     <bool>true</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="left_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>680</y>
      <width>71</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="topleft_label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>54</width>
      <height>111</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeFDiagCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="top_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>650</y>
      <width>54</width>
      <height>131</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="topright_label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>650</y>
      <width>54</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeBDiagCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="right_label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>660</y>
      <width>54</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeHorCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttomright_label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>670</y>
      <width>54</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeFDiagCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttom_label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>680</y>
      <width>54</width>
      <height>121</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeVerCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="buttomleft_label">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>650</y>
      <width>54</width>
      <height>141</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>SizeBDiagCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>buttomright_label</zorder>
   <zorder>topleft_label</zorder>
   <zorder>right_label</zorder>
   <zorder>buttom_label</zorder>
   <zorder>top_label</zorder>
   <zorder>topright_label</zorder>
   <zorder>left_label</zorder>
   <zorder>buttomleft_label</zorder>
  </widget>
 </widget>
 <resources>
  <include location="logos.qrc"/>
 </resources>
 <connections/>
</ui>
